specification/network/resource-manager/Microsoft.Network/stable/2020-06-01/customIpPrefix.json

```diff
@@ -0,0 +1,374 @@
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        }
+      },
+      "get": {
+        "tags": [
+          "CustomIPPrefixes"
+        ],
+        "operationId": "CustomIPPrefixes_Get",
+        "description": "Gets the specified custom IP prefix in a specified resource group.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "customIpPrefixName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the custom IP prefix."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "$expand",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Expands referenced resources."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns the resulting CustomIPPrefix resource.",
+            "schema": {
+              "$ref": "#/definitions/CustomIPPrefix"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get custom IP prefix": {
+            "$ref": "./examples/CustomIpPrefixGet.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "CustomIPPrefixes"
+        ],
+        "operationId": "CustomIPPrefixes_CreateOrUpdate",
+        "description": "Creates or updates a custom IP prefix.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "customIpPrefixName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the custom IP prefix."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CustomIPPrefix"
+            },
+            "description": "Parameters supplied to the create or update custom IP prefix operation."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "201": {
```

Comment:
Will you also add the response code of 202?

Reply:
nope, per NRP contract, GETs and PUTs only respond with OK or CREATED